finagle-redis/src/it/scala/com/twitter/finagle/redis/commands/streams/StreamsClientIntegrationSuite.scala

```diff
@@ -0,0 +1,251 @@
+package com.twitter.finagle.redis.commands.streams
+
+import com.twitter.concurrent.ThreadPoolScheduler
+import com.twitter.conversions.time._
+import com.twitter.finagle.redis.{RedisClientTest, XPendingAllReply, XPendingRangeReply}
+import com.twitter.finagle.redis.tags.{ClientTest, RedisTest}
+import com.twitter.finagle.redis.util.BufToString
+import com.twitter.io.Buf
+import com.twitter.util.{Await, CountDownLatch, Promise}
+import org.scalatest.Inside
+
+final class StreamsClientIntegrationSuite extends RedisClientTest with Inside {
+  val TIMEOUT = 5.seconds
```

Comment:
ffti - we typically use something like `def await[T](f: Future[T], 5.seconds): T` to save a bit more typing